### ISSUE TITLE
Use keras preprocessing layers for preprocessors

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#879]
Image augmentation: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/preprocessing/image_preprocessing.py
normalization layer: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/preprocessing/normalization.py
text vectorization layer: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/preprocessing/text_vectorization.py
